Identity/PresentationLayer/Controllers/BlockingsController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using BusinessLayer.DTOs;
+using BusinessLayer.Extentions;
+using BusinessLayer.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace PresentationLayer.Controllers
+{
+
+    [ApiController]
+    [Route("api/blocking")]
+    public class BlockingsController : ControllerBase
+    {
+
+        private readonly BlockingsService _blockingsService;
+
+        public BlockingsController(BlockingsService blockingsService)
+        {
+            _blockingsService = blockingsService;
+        }
+
+        [HttpGet]
+        [Authorize]
+        public async Task<ActionResult<List<BlockingDto>>> GetBlockingsAsync()
+        {
+            var userId = User.GetUserId();
+            return Ok(await _blockingsService.GetBlockingsAsync(userId));
+        }
+
+        [HttpPost]
+        [Authorize]
+        public async Task<ActionResult> AddBlockingAsync([FromBody] int bid)
+        {
+            var userId = User.GetUserId();
```

Comment:
in get endpoint you included id in route, why did you decide to include id in body in this endpoint?